apps/api/src/app/activity/dtos/shared.dto.ts

```diff
@@ -2,12 +2,20 @@ import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
 import { IsIn, IsOptional, IsString } from 'class-validator';
 
 export enum WorkflowRunStatusDtoEnum {
-  SUCCESS = 'success',
+  PROCESSING = 'processing',
+  COMPLETED = 'completed',
   ERROR = 'error',
+}
+
+export enum WorkflowRunDeliveryLifecycleStatusDtoEnum {
```

Comment:
This one is a very long one to read 😅 Would suggest just `DeliveryLifecycleStatusDtoEnum`